major_event_log/feeds.py

```diff
@@ -10,23 +11,95 @@
 class MajorEventLogFeed(Atom1Feed):
     """Custom Atom Feed for Event objects.
 
-    The only change between this custom feed and the base Atom Feed
-    is that this feed specifies its mime_type as 'application/xml'.
+    The only changes between this custom feed and the base Atom Feed
+    are that this feed specifies its mime_type as 'application/xml',
+    pagination is added, the "alternate" link is removed.
     """
+
     mime_type = 'application/xml'
```

Comment:
This file (and perhaps other code in this repo) has a mix of single and double quotes. Per our DPU Code_style_guide, we should "use single quoted strings (unless the string actually contains single quote characters, then use double)."